functions/src/send-reminders.ts

```diff
@@ -68,8 +106,9 @@ export const sendCampaignStartsReminder = functions.firestore
     });
 
     const message = {
-      from: config.features.reminders.voting_campaign_starts.sender,
-      to: config.features.reminders.voting_campaign_starts.recipient,
+      from: config.features.reminders.voting_campaign_ends.sender,
+      to: config.features.reminders.voting_campaign_ends.recipient,
```

Comment:
Looks like there is a mistake here (`voting_campaign_ends` instead of `voting_campaign_starts`).